src/turner.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <cmath>
+#include <limits>
+#include <ostream>
+
+namespace turner {
+
+#define UNUSED(x) (void)(x)
+
+//
+// Constants
+//
+
+#undef INFINITY
+
+static constexpr float PI = static_cast<float>(M_PI);
+static constexpr float PI2 = static_cast<float>(2 * M_PI);
+static constexpr float INFINITY = std::numeric_limits<float>::max();
+// TODO: Check how to define fat planes properly.
+static constexpr float EPS = 0.00001f;
+
+inline bool is_eps_zero(float a) { return std::abs(a) < EPS; }
```

Comment:
No, the plan would be to move everything to src, when we are done with porting. lib does not make any sense, since it is not a lib at all.